addons/aircraft/CfgAmmo.hpp

```diff
@@ -50,16 +50,27 @@ class CfgAmmo {
         timeToLive = 40;
     };
 
-    // helper projectiles to simulate a rof > fps
-    class ACE_Gatling_30mm_HE_Plane_CAS_01_Deploy: Gatling_30mm_HE_Plane_CAS_01_F {
-        simulation = "shotSubmunitions";
-        triggerTime = 0;
-        submunitionAmmo = "ACE_Gatling_30mm_HE_Plane_CAS_01_Sub";
-        submunitionConeType[] = {"custom", {{0,0}, {0,0}, {0,0}} };
+    // GAU-8 Mix ammo (AP&HE)
+    class ACE_GAU8_30mm_Mix_Plane_CAS_01_F: Gatling_30mm_HE_Plane_CAS_01_F {
+        laserLock = 1;
+        hit = 300;
+        indirectHit = 100;
+        indirectHitRange = 3.5;
```

Comment:
For reference, the 120mm HE has a hit value of 220 and an indirectHit of 60.
The old indirectHit value was 4.
Are these values sane for a 30mm gatling gun?